rio_tiler/io/rasterio.py

```diff
@@ -176,10 +161,13 @@ def _get_descr(ix):
         else:
             nodata_type = "None"
 
+        crs_string = (
+            f"EPSG:{self.crs.to_epsg()}" if self.crs.to_epsg() else self.crs.to_wkt()
```

Comment:
hmm why not opt instead to use a urn/uri rather than attempt a epsg conversion here? those should work with the other authorities and be relatively compact. Failing through to wkt makes sense but also think wkt2_2019 should be preferred as it's more verbose and complete.